images/twitter.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter-page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="grid page">
+        <!-- left bar -->
+        <div class="left-side-bar">
+            <div class="left-menu">
+                <div class="twitter-logo">
+                    <img src="https://www.bing.com/th?id=OIP.VqfuR3kFDkCEND_MnTnZlAHaHa&w=109&h=106&c=8&rs=1&qlt=90&o=6&dpr=2&pid=3.1&rm=2" alt="twitter-logo">
+                </div>
+                <div class="icon">
+                    <img src="https://th.bing.com/th/id/OIP.enl8cyLPjHeaRZiSh9YNQQHaHc?w=124&h=150&c=7&r=0&o=5&dpr=2&pid=1.7" alt="icon">
+                    <div class="blue-home"><strong>Home</strong></div>
+                    </div>
+                <div class="icon">
+                    <img src="https://th.bing.com/th/id/OIP.V-c35hiAW715PW5zj8vVBQHaHa?w=163&h=180&c=7&r=0&o=5&dpr=2&pid=1.7" alt="icon">
+                    <div class="hashtag"><strong>Explore</strong></div>
+                    </div>
+                <div class="icon"><strong>Notifications</strong></div>
+                <div class="icon"><strong>Messages</strong></div>
+                <div class="icon"><strong>Bookmarks</strong></div>
+                <div class="icon"><strong>Lists</strong></div>
+                <div class="icon"><strong>Profile</strong></div>
+                <div class="icon"><strong>More</strong></div>
+
+            </div>
+            <div class="tweet-button">
+                <a href="https://twitter.com/" class="social-link Twitter">Tweet</a>
+
+            </div>
+            <div class="name">@SaraBeazley</div>
+        </div>
+        <!-- Main feed -->
```

Comment:
great use of commenting, very easy to read your work, mine is such a mess haha...